dask/dataframe/rolling.py

```diff
@@ -403,10 +402,6 @@ def apply(
         if has_keyword(meta.apply, "engine"):
             # PANDAS_GT_100
             compat_kwargs = dict(engine=engine, engine_kwargs=engine_kwargs)
```

Comment:
```suggestion
            compat_kwargs = dict(engine=engine, engine_kwargs=engine_kwargs)
```

Reply:
not sure if ``has_keyword(meta.apply, "engine")`` always tests true on pandas 1.0 though? Should test

Reply:
Yeah that's why I left it. If someone can let me know definitively one way or the other, can update